<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Horus Commander</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/img/logo.png</normaloff>:/img/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
      <widget class="QWidget" name="tMain">
       <attribute name="icon">
        <iconset resource="../img.qrc">
         <normaloff>:/img/main.png</normaloff>:/img/main.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>2</number>
        </property>
        <item>
         <widget class="QListWidget" name="clientList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tLog">
       <attribute name="icon">
        <iconset resource="../img.qrc">
         <normaloff>:/img/log.png</normaloff>:/img/log.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Log</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QToolButton" name="missionPrev">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/previous.png</normaloff>:/img/previous.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="missionRun">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/play.png</normaloff>:/img/play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="missionStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/stop.png</normaloff>:/img/stop.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="missionNext">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/next.png</normaloff>:/img/next.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="missionListEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/MLE.png</normaloff>:/img/MLE.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="serverState">
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/serverOff.png</normaloff>:/img/serverOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLeft">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="timeBar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="missionTimeEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/img/timer_clock.png</normaloff>:/img/timer_clock.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Configuration</string>
    </property>
    <addaction name="actionConfigurationPrimary"/>
    <addaction name="actionConfigurationMain"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Online_help"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/minimize.png</normaloff>:/img/minimize.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Minimize</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/show.png</normaloff>:/img/show.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Show</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionConfigurationPrimary">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Primary</string>
   </property>
   <property name="shortcut">
    <string>Alt+Shift+C</string>
   </property>
  </action>
  <action name="actionConfigurationMain">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/settingsAdvanced.png</normaloff>:/img/settingsAdvanced.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Main</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="action_Online_help">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Online help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../img.qrc">
     <normaloff>:/img/info.png</normaloff>:/img/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
